sunpy/io/tests/test_cdf.py

```diff
@@ -23,3 +28,20 @@ def test_read_cdf():
     assert col.unit == u.Unit("1 / (cm2 MeV s sr)")
     # Check that fillvals are replaced by NaN
     assert np.sum(np.isnan(col)) == 189
+
+
+@pytest.mark.remote_data
+def test_read_psp_data():
+    # This was a failing example provided by
+    # https://github.com/sunpy/sunpy/issues/7565
+    dataset = 'PSP_SWP_SPI_SF00_L3_MOM'
+    trange = a.Time('2023-03-14', '2023-03-15')
+    result = Fido.search(trange, a.cdaweb.Dataset(dataset))
+    downloaded_files = Fido.fetch(result)
+
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=SunpyUserWarning)
```

Comment:
You can not catch with with pytest.warns?

Reply:
can not understand

Reply:
Maybe this will help https://docs.pytest.org/en/stable/how-to/capture-warnings.html ?

Reply:
done it through by pytest.warns.

Reply:
Can you match part of the message?

Reply:
Done i guess so.!

Reply:
I dont see the message match?